tools/blockbuster/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: blockbuster
+owner: rnateam
+description: Blockbuster detects blocks of overlapping reads using a gaussian-distribution approach.
+homepage_url: http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html
+long_description: Once short read sequences are mapped to a reference genome, one will face the problem of dividing consecutive reads into blocks to detect specific expression patterns. Due to biological variability and sequencing inaccuracies, the read arrangement does not always show exact block boundaries. The blockbuster tool automatically assigns reads to blocks and gives a unique chance to actually see the different origins where the short reads come from.  http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html
+remote_repository_url: https://github.com/bgruening/galaxytools/tree/master/tools/blockbuster
+type: unrestricted
+categories:
```

Comment:
Actually, the reason seems to be that Anika copied the `+` in the beginning of the line as well.